src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/collectors/NodeStatsAllShardsMetricsCollector.java

```diff
@@ -50,8 +50,8 @@
     private static final int KEYS_PATH_LENGTH = 2;
     private static final Logger LOG = LogManager.getLogger(NodeStatsAllShardsMetricsCollector.class);
     private HashMap<ShardId, IndexShard> currentShards;
```

Comment:
Let's add documentation for these 3 HashMaps.

Reply:
done